manifests/early-starts.yaml

```
@@ -10,6 +10,7 @@ description: Real simple. Start the same way, in the same place, but earlier. Wh
   starts, or to prep yourself well for the main plot, or just to play for years without
   having to worry about the plot.
 url: https://codeberg.org/jmelesky/endless-sky-early-starts/archive/v0.999.zip
+iconUrl: https://codeberg.org/jmelesky/endless-sky-early-starts/raw/branch/main/icon.png
```

Version this URL, please:
```suggestion
iconUrl: https://codeberg.org/jmelesky/endless-sky-early-starts/raw/tag/v0.999/icon.png
```
And likewise in the autoupdate further down:
```
  iconUrl: https://codeberg.org/jmelesky/endless-sky-early-starts/raw/tag/$version/icon.png
```

(This will only work once you create a new release, as the old didn't include the icon)